src/ebay-menu/menu.tsx

```diff
@@ -87,7 +88,7 @@ const EbayMenu: FC<Props> = ({
     }, [checkedIndexes])
 
     return (
-        <span {...rest} className={classNames(className, 'menu')}>
+        <div {...rest} className={classNames(className, 'menu')}>
```

Comment:
We need to be able to support both div and span. By default it will be a span, but we pass the other `baseEl` element as a prop. (We do something similar on  [base dialog](https://github.com/eBay/ebayui-core-react/blob/bf552d6611c82d22368f3cbdfb4b3ffe43b9c5d1/src/ebay-dialog-base/components/dialogBase.tsx#L53))

Reply:
with span remember we noticed that there was display:none from menu.css

Reply:
added support for span and div
https://github.com/eBay/ebayui-core-react/pull/231/commits/a53baf4a9ae5347f602582c94fbeb18c35baef12